vaadinfrontend/frontend/themes/datamanager/components/card.css

```diff
@@ -227,3 +226,74 @@
   list-style-type: none;
   display: contents;
 }
+
+/* Support section formatting and components */
+
+.experiment-item-collection {
+  display: grid;
+  row-gap: 1rem;
+  grid-auto-rows: max-content;
+}
+
+/* Experiment item */
+
+.experiment-item {
+  display: block;
+  height: 100px;
+  width: 100%;
+}
```

Comment:
I'd recommend setting against specific height or widths.
 Otherwise it's hard to account for different screen sizes and varying content. 
![Bildschirmfoto 2023-06-19 um 19 28 14](https://github.com/qbicsoftware/data-manager-app/assets/29627977/beeefe0d-d7bf-4af3-8cb4-cdcb794ecf7d)
```suggestion
.experiment-item {
  display: flex;
  width: 100%;
  flex-direction: column;
}
```

Reply:
but then maybe a `min-heigth`? otherwise it does not fit in the UI design